radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -147,6 +147,17 @@ elseif(PCB STREQUAL X7)
     add_definitions(-DRADIO_TPROV2)
     add_definitions(-DMANUFACTURER_JUMPER)
     set(ENABLE_SERIAL_PASSTHROUGH ON CACHE BOOL "Enable serial passthrough")
+  elseif(PCBREV STREQUAL TPROS)
+    set(DEFAULT_INTERNAL_MODULE MULTIMODULE CACHE STRING "Default internal module")
```

Comment:
MPM or CRSF? Or is this another "will be wrong half of the time" gotchas? 

Other than that, LGTM, my TPro S still boots and works fine, rest of the defaults were correct other than internal RF.

Reply:
My bad obviously :)

Initially it will be ELRS only, but may be proposed later with Multi should the market express a need for it.